.cirrus.yml

```diff
@@ -403,7 +403,7 @@ task:
     - cmake --build build --config RelWithDebInfo -- -property:UseMultiToolTask=true;CL_MPcount=5
   check_script:
     - '%x64_NATIVE_TOOLS%'
-    - ctest --test-dir build -j 5
+    - ctest -C RelWithDebInfo --test-dir build -j 5
```

Comment:
Is this change necessary? Shouldn't that be the default already?

Reply:
> Is this change necessary? Shouldn't that be the default already?

It does. Otherwise, on Windows, `ctest` complains:
```
Test not available without configuration.  (Missing "-C <config>"?)
```